vision/benchmarks/accuracy/bench_module.py

```diff
@@ -352,26 +352,33 @@ def bench_module_accuracy(folder: str, quiet_output: bool = False, draw_centers:
                         "CRASH:",crash,
                     )
                 
+
+                marked_image = np.copy(image)
```

Comment:
This can be moved inside the an if statement so that it is only copied if needed. Example:

`if draw_centers or (draw_hough_circles and tester.location.circles is not None):`